app/services/message_service.rb

```diff
@@ -0,0 +1,63 @@
+class MessageService
```

Comment:
As recommended by Bukola, a service is better than a helper because a helper makes the methods available globally (even where they are not needed e.g views) while services limits it to the controllers.